File: java-spring/eventstore-examples-java-web/src/main/java/net/chrisrichardson/eventstore/javaexamples/banking/web/main/BankingMain.java
package net.chrisrichardson.eventstore.javaexamples.banking.web.main;

import net.chrisrichardson.eventstore.javaexamples.banking.web.BankingWebConfiguration;
import org.springframework.boot.SpringApplication;

public class BankingMain {

  public static void main(String[] args) {
    SpringApplication.run(BankingWebConfiguration.class, args);
  }
}

File: scala-spring/eventstore-examples-main/src/main/scala/net/chrisrichardson/eventstore/examples/bank/web/main/BankingMain.scala
package net.chrisrichardson.eventstore.examples.bank.web.main

import net.chrisrichardson.eventstore.examples.bank.web.BankingWebAppConfiguration
import org.springframework.boot.SpringApplication

object BankingMain  {

  def main(args: Array[String]) : Unit = SpringApplication.run(classOf[BankingWebAppConfiguration], args :_ *)

}
